### DISCUSSION TITLE
enable wifi

### DISCUSSION THREAD

**Discussion by hazerflac7** [Ysurac/openmptcprouter#2466] [Q&A]
I installed openmptc on a pc and everything is up and running. But I don't know how to enable wifi on my pc. The device always remain inactive and not associated with anything. It will scan and see other network so I know its working. What setting do I need?

**Comment by yannick2801** [comment]
have you resolved your problem ? BR

i have the same problem on ROCK pi 4 SE : 
![image](https://github.com/Ysurac/openmptcprouter/assets/97221644/b00ecf73-77b4-4607-9694-5553c49263ee)

![image](https://github.com/Ysurac/openmptcprouter/assets/97221644/a20015c4-b8b2-4ab7-92f9-bd10f0d31049)

Yannick

**Reply by Trent63** [reply]
first, create a Client network.
Then you can create an access point. And delete the Client. I suffered for several evenings before it worked out

**Reply by yannick2801** [reply]
i managed to get it worked like on RASPBERRY.
BR
Yannick

**Reply by Trent63** [reply]
Can you share a ready-made image for ROCK PI? I can't put it together((
